src/android_app_plugin/kolibri_plugin.py

```diff
@@ -73,11 +74,10 @@ def update(self, job, orm_job, state=None, **kwargs):
             # avoid passing integers that are too large
             # PROGRESS_LIMIT gives sufficient precision for a % progress calculation
             if total_progress > PROGRESS_LIMIT:
-                progress = progress // total_progress * PROGRESS_LIMIT
+                progress = PROGRESS_LIMIT * progress // total_progress
```

Comment:
Oh yes... the previous version would always go to 0... I guess the math didn't check out!

Reply:
I originally had `math.floor(...)` and tested it with that, then I thought "oh yeah there's integer division `//` which is simpler" (self-sabotage)